src/WordPressCodeExtractor.php

```diff
@@ -91,7 +91,17 @@ public static function fromFile( $file, Translations $translations, array $optio
 	 */
 	public static function fromDirectory( $dir, Translations $translations, array $options = [] ) {
 		static::$dir = $dir;
-		static::fromFile( static::getFilesFromDirectory( $dir ), $translations, $options );
+
+		$files = static::getFilesFromDirectory( $dir );
+
+		try {
+			if ( ! empty( $files ) ) {
+				static::fromFile( $files, $translations, $options );
```

Comment:
Unrelated to this PR, but makes me wonder whether `public static function fromFile( $file, ... )` should not be named `public static fromFiles( $files, ... )` instead (plural vor name and argument), as it loops over multiple files.
Alternatively, the loop could be extracted out of `fromFile()` and put into the surrounding code. Might make more sense, conceptually. In that case, the above would turn from an `if` into a simple `foreach`.

Reply:
I totally agree that it's not ideal.

Just note that this method implements `ExtractorInterface::fromFile()` where the doc block says `@param array|string $file A path of a file or files`. The abstract `Extractor` class it extends does the same, so I tried to keep the same behavior.

Reply:
Okay, let's keep it as is for now then.